src/cascadia/TerminalSettingsModel/Resources/en-US/Resources.resw

```diff
@@ -391,4 +391,16 @@
   <data name="OpenWindowRenamerCommandKey" xml:space="preserve">
     <value>Rename window...</value>
   </data>
-</root>
+  <data name="InboxWindowsConsoleAuthor" xml:space="preserve">
+    <value>Microsoft Corporation</value>
+    <comment>Paired with `InboxWindowsConsoleName`, this is the application author... which is us: Microsoft.</comment>
+  </data>
+  <data name="InboxWindowsConsoleName" xml:space="preserve">
+    <value>Windows Console Host</value>
+    <comment>Name describing the usage of the classic windows console as the terminal UI. (`conhost.exe`)</comment>
+  </data>
+  <data name="TerminalVersionPattern" xml:space="preserve">
+    <value>{0}.{1}.{2}.{3}</value>
+    <comment>Pattern for representing a version. 0 is major. 1 is minor. 2 is build. 3 is revision.</comment>
+  </data>

```

Comment:
Do we need to localize this?

Reply:
I don't know for sure. I thought maybe other cultures use a different separator for version numbers? I don't know.

Reply:
Meh. Don't bother localizing this one. CascadiaSettings::ApplicationVersion uses a printf with %d.%d.%d.%d.